Core/MIPS/ARM/ArmAsm.cpp

```diff
@@ -70,9 +71,84 @@ namespace MIPSComp {
 
 using namespace ArmJitConstants;
 
-void ArmJit::GenerateFixedCode()
-{
-	enterCode = AlignCode16();
+void ArmJit::GenerateFixedCode() {
+	const u8 *start = GetCodePtr();
+
+	// LR == SCRATCHREG2....  Maybe we could find some other register to use to avoid having to
+	// push LR?
```

Comment:
I wonder... maybe we just enforce that other regs must be flushed?  As long as it's always after a FlushAll (might already be), we could use other regs normally used for alloc.

-[Unknown]